app/util/notifications/methods/index.ts

```diff
@@ -161,8 +161,9 @@ export function getNetwork(chain_id: HalRawNotification['chain_id']) {
   return ChainId[chain_id];
 }
 
-export const isNotificationsFeatureEnabled = () =>
-  process.env.MM_NOTIFICATIONS_UI_ENABLED;
+export const isNotificationsFeatureEnabled = () => {
+  return !!(process.env.MM_NOTIFICATIONS_UI_ENABLED === 'true');
```

Comment:
```suggestion
  return process.env.MM_NOTIFICATIONS_UI_ENABLED === 'true';
```

The `!!` is redundant.

Reply:
Implemented.